src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -11,5 +11,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
```

Comment:
we should probably delete `PREFIX_PHONE` and `PREFIX_EMAIL` in the future as well